rust_src/src/eval.rs

```diff
@@ -232,7 +232,7 @@ pub fn function(args: LispCons) -> LispObject {
                         // dynamically.
 
                         let docstring = unsafe { eval_sub(car(tail)) };
-                        docstring.as_string_or_error();
```

Comment:
This only verifying that `docstring` is a string right? The case hides the purpose.

Reply:
Changed this one to be clearer.